.github/workflows/nightly.yml

```diff
@@ -117,6 +122,11 @@ jobs:
         - 15672:15672
 
     steps:
+    - name: Check access to GitHub secrets
+      run: echo "Pull request that trigger this workflow need to be on upstream repo to access GitHub secrets."; exit 1
+      # if a pull request is merged from a fork
+      if: github.event_name == 'pull_request' && github.event.pull_request.head.repo.owner.login != 'aiidateam'
+
```

Comment:
Not needed here, the Slack message step is already gated by `&& env.SLACK_WEBHOOK != null` so the nightly can run on forks normally (and works now)

Reply:
true it is just the slack notification which is not run